composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.5",
```

Comment:
Oh sorry! My bad! Didn't see it, seems to be real... :/ https://travis-ci.org/Ph3nol/NotificationPusher/builds/77487072

Thanks for this!